src/book/03-guides/01-main-codebase/03-recording-playback.mdx

```diff
@@ -0,0 +1,120 @@
+
+2. Set up the configuration file. This involves making changes to the [`DataLogging.yaml`](https://github.com/NUbots/NUbots/tree/master/module/support/logging/DataLogging/data/config) file. Add in the message you want to log, specifying `true`.
+
+   In this example, the sensors message already exists in the file (`message.input.Sensors`). Set the boolean to `true` next to this message.
+
+   If you would like to log a different message:
+
+1. Find the `.proto` file that defines that message, such as [`Sensors.proto`](https://github.com/NUbots/NUbots/blob/master/shared/message/input/Sensors.proto).
+   2. Find what the `package` is. For `Sensor.proto`, it is `package message.input`.
+   3. Piece this together with the message name (`message Sensors` in the `.proto` file) to get `message.input.Sensors`
+   4. Add this to the configuration file with the other messages.
+
+   You can enable multiple messages at once.
+
+3. Build the code and install it to a robot. To find out how to do this, visit the [Getting Started](/guides/nubots/getting-started) page.
+
+4. Run the binary. In our example, this would be `./sensortest`.
+
+5. Stop the binary when you have finished recording by pressing <kbd>CTRL</kbd> + <kbd>c</kbd>.
+
+6. Move the newly created NBS file to a computer for safe storage.
+   1. The NBS file will be located on the robot in the `log/role_name/` folder. The name will correspond to the time and date it was created, according to the robot's clock. Make note of the name. In our example, this may be `log/sensortest/18072020T082100.nbs`.
+   2. Copy the file across. On the destination computer, run `scp nubots@<address>:log/role_name/file_name path/to/destination`. In our example, we may want to run `scp nubots@<address>:log/sensortest/18072020T082100.nbs .`. The file will then be copied to the folder you ran the command in. `<address>` corresponds to the address of the robot.
+
+## Extracting Data
+
+The NBS file is not easily understood by humans. Certain tools in the NUbots codebase convert NBS files to file formats that are easier for us to understand.
+
+The tools are run with the **b** script in the NUbots codebase. [NBS tools](https://github.com/NUbots/NUbots/tree/master/tools/nbs) have the argument `nbs`.
+
+Tools are executed by running
+
+```sh
+./b nbs <tool_name> <nbs_file_name> <arguments>
+```
+
+### Stats
+
+Decodes an NBS file into a series of file statistics.
+
+| Argument            | Definition                                                        |
+| ------------------- | ----------------------------------------------------------------- |
+| --message-timestamp | Uses the message's `.timestamp` field as the time of the message. |
```

Comment:
I'd recommend making the arguments inline code blocks, same with the arguments for the rest of the tools on this page.

```suggestion
| `--message-timestamp` | Uses the message's `.timestamp` field as the time of the message. |
```